src/Service/Automations/HTTPDefault.hs

```diff
@@ -0,0 +1,45 @@
+module Service.Automations.HTTPDefault
```

Comment:
I don't really like this boilerplate just to trigger a message but I'm not sure how else to do it relatively cleanly